webob/response.py

```diff
@@ -1001,6 +1002,12 @@ def set_cookie(self, name=None, value='', max_age=None,
            in the cookie, if it's ``False``, the ``HttpOnly`` flag will not
            be sent in the cookie.
 
+        ``samesite``
+
+          A string representing the ``SameSite`` attribute of the cookie or
+          ``None``. If samesite is ``None`` no ``SameSite`` value will be sent
```

Comment:
Are the various casing and markups of samesite and SameSite intentional?

Also are there two contexts or one for samesite/SameSite? Each context should use consistent markup.